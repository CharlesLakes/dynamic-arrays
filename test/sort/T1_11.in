10
31 98 83 86 17 35 12 57 90 32