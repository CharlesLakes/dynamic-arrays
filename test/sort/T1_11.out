12 17 31 32 35 57 83 86 90 98